core/vm/eips.go

```diff
@@ -192,3 +193,32 @@ func opPush0(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]by
 	scope.Stack.push(new(uint256.Int))
 	return nil, nil
 }
+
+func enable5656(jt *JumpTable) {
+	jt[MCOPY] = &operation{
+		execute:     opMemoryCopy,
+		constantGas: GasFastestStep,
+		dynamicGas:  gasMemoryCopy,
+		minStack:    minStack(3, 0),
+		maxStack:    maxStack(3, 0),
+		memorySize:  memoryMemoryCopy,
+	}
+}
+
+func opMemoryCopy(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
```

Comment:
```suggestion
func opMcopy(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
```
Just convention to use the op mnemonic in the impl name.